src/Microsoft.NET.Sdk.Functions/Microsoft.NET.Sdk.Functions.csproj

```diff
@@ -1,18 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.5;net46</TargetFrameworks>
-    <PackageVersion>1.0.2</PackageVersion>
+    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
+    <PackageVersion>1.0.3</PackageVersion>
     <BuildOutputTargetFolder>tools</BuildOutputTargetFolder>
     <AssemblyOriginatorKeyFile>PublicKey.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>False</SignAssembly>
     <DelaySign>True</DelaySign>
   </PropertyGroup>
 
   <ItemGroup>
-    <Content Include="Targets\netstandard1.0\**\*">
```

Comment:
you can leave all the netstandard1.0 as is. This just means that the targets are netstandard1.0 compatible.